src/parse/utilities.ts

```diff
@@ -15,8 +15,8 @@ export function getIndentation(line: string): string {
  */
 export function preprocessLines(lines: string[]): string[] {
     return lines
-        .map(line => line.trim())
-        .filter((line) => !line.startsWith("#"));
+        .filter((line) => !line.trim().startsWith("#"))
```

Comment:
We can't do this unfortunately because it will cut off strings with # in them. See previous discussion here: https://github.com/NilsJPWerner/autoDocstring/pull/144